examples/user_echo_webhook.py

```diff
@@ -6,9 +6,8 @@
                  bot will create a webhook to "impersonate"
                  specified user, so to speak."""
 
-intents = (
-    disnake.Intents.all()
-)  # Not limiting the permissions, since this is an example - set this accordingly.
+# Not limiting the permissions, since this is an example - set this accordingly.
```

Comment:
intents

Reply:
fixed in 89a5c65121de57610b8116404aa94c288f5e53b0